include/defs.h

```diff
@@ -51,3 +51,12 @@
  */
 #define AOL_FILENAME "aol"
 
+/* xXx DEFINE=OL_SUCCESS xXx
+ * xXx DESCRIPTION=Used to indicate an operation went according to plan.
+ */
+#define OL_SUCCESS 0
```

Comment:
I believe we mostly decided to avoid this: https://github.com/infoforcefeed/OlegDB/issues/76

I guess now we don't have the granular error codes, but I'm kind of planning for expansion here with the `ol_error` function instead.